### PR TITLE
Clean up verbose code

### PR DESCRIPTION
**Pull request by janus**
Cleaned up some codes inside althea_kernel_interface folder .

### CODE REVIEW

File: althea_kernel_interface/src/ip_route.rs [hunk 1]
```diff
@@ -6,15 +6,14 @@ use failure::Error;
...
-            .run_command("ip", &["route", "list", "default"])
+        let output = self.run_command("ip", &["route", "list", "default"])
             .unwrap();
 
         let stdout = String::from_utf8(output.stdout).unwrap();
 
         let mut def_route = Vec::new();
 
-        let mut found = false;
+        //let mut found = false;
```
Comment (kingoflolz) [inline althea_kernel_interface/src/ip_route.rs]: delete the code if it is not needed

File: althea_kernel_interface/src/ip_route.rs [hunk 2]
```diff
@@ -53,9 +46,9 @@ impl KernelInterface {
     }
 
     fn set_default_route(&self, route: &Vec<String>) -> Result<(), Error> {
-        let mut def_route_ref: Vec<&str> = vec!["route", "add"];
+        let mut def_route_ref: Vec<&str> = vec!["route", "add", "default"];
 
-        def_route_ref.push("default");
+        //def_route_ref.push("default");
```
Comment (kingoflolz) [inline althea_kernel_interface/src/ip_route.rs]: same here

File: althea_kernel_interface/src/lib.rs [hunk 3]
```diff
@@ -100,6 +100,7 @@ impl CommandRunner for LinuxCommandRunner {
     }
 
     fn set_mock(&self, _mock: Box<FnMut(String, Vec<String>) -> Result<Output, Error> + Send>) {
+        println!("unimplemented!()");
```
Comment (kingoflolz) [inline althea_kernel_interface/src/lib.rs]: Don't think this is needed, the unimplemented macro tells us that it was hit

File: althea_kernel_interface/src/open_tunnel.rs [hunk 4]
```diff
@@ -9,7 +9,7 @@ fn to_wg_local(ip: &IpAddr) -> IpAddr {
     match ip {
         &IpAddr::V6(ip) => {
             let mut seg = ip.segments();
-            assert_eq!((seg[0] & 0xfd00), 0xfd00);
+            assert_eq!((seg[0] & 0xffff), 0xfd00);
```
Comment (kingoflolz) [inline althea_kernel_interface/src/open_tunnel.rs]: we only want to check the first 9 bits, not the whole first 16

File: althea_kernel_interface/src/ip_route.rs (hunk 1 above)
Comment (jkilpatr) [inline althea_kernel_interface/src/ip_route.rs]: I like this change you can just delete this line though, no need to leave it commented out.

File: althea_kernel_interface/src/stats.rs [hunk 5]
```diff
@@ -5,62 +5,51 @@ use std::fs::File;
 use std::io::Read;
 
 impl KernelInterface {
-    pub fn get_proc_stat(&self) -> Result<String, Error> {
-        debug!("getting proc stat");
-        let mut f = File::open("/proc/stat")?;
-
+    fn get_stat(&self, path_str: &str) -> Result<String, Error> {
```
Comment (kingoflolz) [inline althea_kernel_interface/src/stats.rs]: maybe this should be read_file ?

File: althea_kernel_interface/src/ip_route.rs (hunk 2 above)
Comment (jkilpatr) [inline althea_kernel_interface/src/ip_route.rs]: same thing here.

File: althea_kernel_interface/src/lib.rs (hunk 3 above)
Comment (jkilpatr) [inline althea_kernel_interface/src/lib.rs]: not needed the unimplemented!() should print something like that for us.

**Comment by jkilpatr** [comment]
looks good, I'll let @kingoflolz review again then I'll merge

**Comment by drozdziak1** [comment]
Yup @janus, like @kingoflolz said you need to run cargo-fmt on your code. Make sure it's up-to-date with `cargo install rustfmt-nightly --force` and then do `cargo fmt` inside the project and then commit and push.

**Comment by jkilpatr** [comment]
https://github.com/althea-mesh/althea_rs/pull/150

cleaned up here, will merge once tested there.

**Pull request closed without merging**